src/GameScore/GameScore.cpp

```diff
@@ -67,41 +69,68 @@ void GameScore::countPointsInHand(Player &player) {
     int carrots = countGoods(*player.getGameHand(), "Carrot");
     int trees = countGoods(*player.getGameHand(), "Tree");
     int wilds = countGoods(*player.getGameHand(), "Wild");
+    
+    // If a computer has wildcards, assign them to the good they have the most of
+    if (player.getPlayerStrategies()->displayCurrentStrategy().compare("Human Player") != 0 && wilds > 0) {
+        // Find the good the player has the most of
+        int maxGood = std::max({rocks, crystals, anvils, carrots, trees});
```

Comment:
clean implementation! love it

Reply:
That's if you call this monster "clean":
```cpp
player.getPlayerStrategies()->displayCurrentStrategy().compare("Human Player") != 0 && wilds > 0
```